libs/platform/core/server/services/page-meta/server-page-meta.service.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  DefaultPageMetaService,
+  ElementAttributes,
+  ElementDefinition,
+} from '@spryker-oryx/core';
+
+export class ServerPageMetaService extends DefaultPageMetaService {
+  protected template = '';
+
+  add(definitions: ElementDefinition | ElementDefinition[]): void {
+    if (!Array.isArray(definitions)) {
+      definitions = [definitions];
+    }
+
+    let stream = ``;
+
+    for (const { name, attrs } of definitions) {
+      if (name === 'html') {
+        this.setHtmlAttributes(attrs);
+
+        continue;
+      }
+
+      if (attrs.text) {
+        stream += `\n<${name}>${attrs.text}</${name}>`;
+
+        continue;
+      }
+
+      stream += `\n<${name}${this.setAttributes(attrs)} />`;
+    }
+
+    this.template = this.template.replace('</head>', `${stream}\n</head>`);
+  }
+
+  setHtmlAttributes(attrs: ElementAttributes): void {
+    this.template = this.template.replace(
```

Comment:
not sure if this one will work on the server app, if used inside some component or service?